api/cluster-config/v1/clusterConfigClient.go

```diff
@@ -23,7 +24,10 @@ func CreateClusterConfigClient(kubeconfig string) (*v1alpha1.CRDClient, error) {
 		return nil, err
 	}
 
-	v1alpha1.AddToRegistry("clusterconfigs", &ClusterConfig{}, &ClusterConfigList{})
+	v1alpha1.AddToRegistry("clusterconfigs", &ClusterConfig{}, &ClusterConfigList{}, nil, schema.GroupResource{
+		Group:    GroupVersion.Group,
+		Resource: "peeringrequests",
```

Comment:
```suggestion
		Resource: "clusterconfigs",
```
a copy and paste refusal I think 😄

Reply:
nice catch!